joomla/joomla-cms
# JFormFieldList shows the global values if present.

Pull Request for Issue #11855.
### Summary of Changes

The list formfield (the one responsible for dropdown selects in forms) will now show the global setting if available.
The field tries to load the component params and checks if the fieldname exists as param. If it does it will show the value.
### Testing Instructions

Check the article settings and see that the global values are now shown ~~behind~~ within the select.
Check if there are any sideeffects in unexpected locations (mainly other forms).
### Documentation Changes Required

~~None.~~
JFormFieldList now supports `useglobal` as an attribute. It will add the option "Use Global" and shows the value if found.

I honestly don't know if that should be included in core or not. I just saw the request and thought I see if I can do it. It feels a bit hacky to me to fiddle with extension params in a library class. Maybe someone even has a better idea.
